Core/Src/bms.c

```diff
@@ -8,10 +8,6 @@
 #include <stdlib.h>
 #include "stdio.h"
 
-#define CAN_QUEUE_SIZE 5
-
-osMessageQueueId_t bms_monitor_queue;
-
 bms_t *bms;
 
 void bms_fault_callback();
```

Comment:
this needed anymore?

Reply:
Yes. It is the callback of the AMS timer that faults the car if the AMS watchdog times out.